.github/policies/scheduledSearches.yml

```diff
@@ -186,7 +186,6 @@ configuration:
           - hourly:
               hour: 3
         filters:
-          - isIssue
```

Comment:
Please update description above to align with AVM repo: `"ITA04 - Label issues and PRs that have been marked as requiring author feedback but have not had any activity for 4 days."`

Reply:
Good catch! I overlooked this. Now updated. Thanks. Please resolve.